backend_windows.go

```diff
@@ -244,6 +246,13 @@ func (w *Watcher) Add(name string) error {
 //
 // Removing a path that has not yet been added returns [ErrNonExistentWatch].
 func (w *Watcher) Remove(name string) error {
+	w.mu.Lock()
+	if w.isClosed {
```

Comment:
Should the Windows isClosed be modified to lock the Mutex, or are there places where that could cause deadlock if we did?

Reply:
Fixed in f8aebb4. Nice!